index.d.ts

```diff
@@ -2,6 +2,33 @@ import {GlobbyOptions} from 'globby';
 import {Options as CpFileOptions} from 'cp-file';
 
 declare namespace cpy {
+	interface SourceFile {
+		/**
+		Path to file.
+		*/
+		readonly path: string,
+
+		/**
+		Resolved path to file.
+		*/
+		readonly resolvedPath: string,
+
+		/**
+		File name.
```

Comment:
Filename is a word.

```suggestion
		Filename.
```

Same with the below.